packages/mui-base/src/Field/Root/FieldRoot.types.ts

```diff
@@ -28,6 +28,16 @@ export interface FieldRootOwnerState {
 }
 
 export interface FieldRootProps extends BaseUIComponentProps<'div', FieldRootOwnerState> {
+  /**
+   * Whether the field is disabled. Takes precedence over the `disabled` prop of the `Field.Control`
+   * component.
+   * @default false
+   */
+  disabled?: boolean;
+  /**
+   * The field's name. Takes precedence over the `name` prop of the `Field.Control` component.
+   */
+  name?: string;
```

Comment:
@michaldudak to avoid changing unrelated components too much, we can do namespace types refactoring in a new PR